core/src/main/java/com/sequenceiq/cloudbreak/core/flow/service/ReactorFlowManager.java

```diff
@@ -135,13 +135,15 @@ public void triggerTermination(Object object) {
         StackDeleteRequest deleteRequest = (StackDeleteRequest) object;
         DefaultFlowContext context = new DefaultFlowContext(deleteRequest.getStackId(), deleteRequest.getCloudPlatform());
         reactor.notify(FlowPhases.TERMINATION.name(), eventFactory.createEvent(context, FlowPhases.TERMINATION.name()));
+        reactor.notify("FLOWCANCEL", eventFactory.createEvent(context, "FLOWCANCEL"));
```

Comment:
The [value](https://github.com/sequenceiq/cloudbreak/pull/1499/files#diff-22b37aa2388a0e062b1364b438e9da15R26) is the event, not the name.